apps/frontend/app/(website)/studio/src/store/snippets.ts

```diff
@@ -108,174 +107,186 @@ const getEditorsFromLS = () => {
   if (!editors) return;
   return;
 };
-export const useSnippetsStore = create<SnippetsState>(
-  persist(
-    (set, get) => ({
-      tabsLimit: 12,
-      getHasReachedTabsLimit: () => get().editors.length >= get().tabsLimit,
-      editors: INITIAL_STATE.editors,
-      addPair: () => {
+export const useSnippetsStore = create<SnippetsState>((set, get) => ({
+  tabsLimit: 12,
+  getHasReachedTabsLimit: () => get().editors.length >= get().tabsLimit,
+  // editors: localStorage.getItem('editors') ? JSON.parse(localStorage.getItem('editors')) : [getSingleTestCase()],
+  addPair: (name?: string) => {
+    set({
+      editors: [
+        ...get().editors,
+        {
+          name:
+            name ||
```

Comment:
`??`

Reply:
nope - we dont want to have empty string here